components/UI/profileCard/profileCard.tsx

```diff
@@ -8,37 +8,48 @@ import React, {
 import styles from "@styles/dashboard.module.css";
 import CopyIcon from "@components/UI/iconsComponents/icons/copyIcon";
 import { CDNImage } from "@components/cdn/image";
-import { useStarkProfile,type Address } from "@starknet-react/core";
+import { useStarkProfile, type Address } from "@starknet-react/core";
 import { minifyAddress } from "@utils/stringService";
 import trophyIcon from "public/icons/trophy.svg";
 import xpIcon from "public/icons/xpBadge.svg";
 import useCreationDate from "@hooks/useCreationDate";
 import shareSrc from "public/icons/share.svg";
 import theme from "@styles/theme";
-import { Tooltip } from "@mui/material";
-import CopyAddress from "@components/UI/CopyAddress"; 
+
 import ProfilIcon from "../iconsComponents/icons/profilIcon";
 import Link from "next/link";
 import SocialMediaActions from "../actions/socialmediaActions";
-import { getTweetLink, writeToClipboard } from "@utils/browserService";
+import { getTweetLink } from "@utils/browserService";
 import { hexToDecimal } from "@utils/feltService";
-import { Url } from "next/dist/shared/lib/router/router";
 import { TEXT_TYPE } from "@constants/typography";
 import Typography from "../typography/typography";
 
-const ProfileCard: FunctionComponent<ProfileCard> = ({
+const EyeIcon = () => (
```

Comment:
Add this icon to `components/UI/iconsComponents/icons` as we do for all other icons